README.md

```diff
@@ -146,9 +146,52 @@ RTSan's algorithm consists of two parts that work together:
 
 # Using RTSan with other compilers
 
-Yes, it's possible. Here's how.
+The recommended way to use RTSan is to use it with LLVM 20 directly, as described elsewhere in this document. The rest of this section describes a hack which may or may not continue to work in the future.
```

Comment:
This is an initial draft of these docs, I need to actually go through this example and make sure it works on both systems. It should be **mostly** right, but I haven't checked it out